### ISSUE TITLE
Use tiddlywiki's sidebar breakpoint in streams/styles

### ISSUE THREAD

**Issue by postkevone** [saqimtiaz/streams#37]
At the moment streams uses fixed values in the styles tiddler:
`@media only screen and (max-width: 600px)`

It would be ideal to use the default sidebar breakpoint value stored in
`$:/themes/tiddlywiki/vanilla/metrics/sidebarbreakpoint`
for better compatibility and customization.

**Comment by saqimtiaz** [comment]
@postkevone interesting idea. The problem is of course that the config tiddler you mentioned is theme specific so wont work for someone with a heavily customized theme.

A more flexible option might be to introduce a new config tiddler to control the width used in that media query.

**Comment by postkevone** [comment]
You are right, perhaps the best solution is to have a config tiddler inside streams itself
